Djikstra/Djikstra/Functions.cpp

```diff
@@ -0,0 +1,135 @@
+#include <vector>
+
+
+#include "Headers.h"
+
+
+namespace Graph
+{
+	bool Graph::node_exist( std::string name)
```

Comment:
Use 4 spaces instead of tabs. You can set it up in the preferences of VS (read more [here](https://docs.microsoft.com/en-us/visualstudio/ide/reference/options-text-editor-all-languages-tabs?view=vs-2017)).